cmd/kubelet-gpu-plugin/node_state.go

```diff
@@ -112,6 +112,12 @@ func (s *nodeState) GetResources() kubeletplugin.Resources {
 					"family": {
 						StringValue: &gpu.FamilyName,
 					},
+					"pci-device-id": {
+						StringValue: &gpu.Model,
+					},
+					"pci-device-address": {
+						StringValue: &gpu.PCIAddress,
+					},
```

Comment:
We have `pciRoot` field already in the pipeline for an upcoming release, let's have these named with same style.
```suggestion
					"pciId": {
						StringValue: &gpu.Model,
					},
					"pciAddress": {
						StringValue: &gpu.PCIAddress,
					},
```